external-authority-sources.md

```diff
@@ -0,0 +1,275 @@
+# External sources Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/integration/externalsources**   
+
+Provide access to the configured external sources. It returns the list of existent external sources.
+
+Example:
+```json
+{
+  "_embedded": {
+    "externalsources": [
+      {
+        "id": "orcid",
+        "name": "orcid",
+        "hierarchical": false,
+        "type": "externalsource",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/orcid/entries"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/orcid"
+          }
+        }
+      },
+      {
+        "id": "ciencia",
+        "name": "ciencia",
+        "hierarchical": false,
+        "type": "externalsource",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/ciencia/entries"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/ciencia"
+          }
+        }
+      },
+      {
+        "id": "my_staff_db",
+        "name": "my_staff_db",
+        "hierarchical": false,
+        "type": "externalsource",
+        "_links": {
+          "entries": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/my_staff_db/entries"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/my_staff_db"
+          }
+        }
+      }
+    ]
+  },
+  "_links": {
+    "self": {
+      "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources"
+    }
+  },
+  "page": {
+    "size": 20,
+    "totalElements": 3,
+    "totalPages": 1,
+    "number": 0
+  }
+}
+```
+
+## Single Authority
+**/api/integration/externalsources/<:authority-name>**
+
+Provide detailed information about a specific external source. The JSON response document is as follow
+```json
+{
+  "id": "orcid",
+  "name": "orcid",
+  "hierarchical": false,
+  "type": "externalsource"
+}
+```
+
+Exposed links:
+* entries: the list of values managed by the external source
+
+## Linked entities
+### external source entries
+**/api/integration/externalsources/<:authority-name>/entries**
```

Comment:
here <:source-name> makes more sense to me than <:authority-name>

Reply:
This is indeed better to avoid confusion. I've updated it accordingly